Sources/RichTextKit/Colors/RichTextColorPicker.swift

```diff
@@ -30,18 +30,28 @@ public struct RichTextColorPicker: View {
     public init(
         icon: Image?,
         value: Binding<Color>,
-        quickColors: [Color] = []
+        quickColors: [Color] = [],
+        type: ColorType = .normal
```

Comment:
This looks great! I actually had something similar previously, but removed it for simplicity. :)